operations/fetch_test.go

```diff
@@ -5,63 +5,35 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/evergreen-ci/evergreen/testutil"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestClone(t *testing.T) {
-	settings := testutil.TestConfig()
-	testutil.ConfigureIntegrationTest(t, settings)
-
-	type testCase struct {
-		opts      cloneOptions
-		isPassing bool
+	opts := cloneOptions{
+		owner:      "evergreen-ci",
+		repository: "sample",
+		revision:   "cf46076567e4949f9fc68e0634139d4ac495c89b",
+		branch:     "main",
 	}
+	opts.rootDir = t.TempDir()
+	assert.NoError(t, clone(opts))
 
-	testCases := map[string]testCase{
-		"SimpleHTTPS": {isPassing: true, opts: cloneOptions{
```

Comment:
Are we able to keep this format and just remove the SimpleHTTPS case? Lmk if I'm missing something about what was changed here.

Reply:
the simpleHTTPS case is still there, it's just the first setup
https://github.com/evergreen-ci/evergreen/pull/8386/files#diff-f3e98349a5ccd629b2b8e1c1c2603cbfe2b619824af2caefccee4b1a270062ecR12-R16